.github/workflows/tests.yml

```diff
@@ -20,11 +20,6 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
-  check:
-    uses: OpenAstronomy/github-actions-workflows/.github/workflows/tox.yml@86823142467dd2afcd1bbac57d3a664cf468eb3b  # v2.1.0
-    with:
-      envs: |
-        - linux: check-style
```

Comment:
The bot is failing to find a pre-commit file: https://results.pre-commit.ci/run/github/488230940/1756233873.mj6g-oCJQFOdUOxSfxk8Qg
and without one we can't drop `check-style`.